controller/proxy-injector/webhook.go

```diff
@@ -74,7 +74,7 @@ func Inject(linkerdNamespace string) webhook.Handler {
 		if ownerRef := resourceConfig.GetOwnerRef(); ownerRef != nil {
 			res, err := k8s.GetAPIResource(ownerRef.Kind)
 			if err != nil {
-				log.Warningf("skipping event for parent %s: %s", ownerRef.Kind, err)
+				log.Debugf("skipping event for parent %s: %s", ownerRef.Kind, err)
```

Comment:
This should change the level to trace. I agree it's a better choice.

```suggestion
				log.Tracef("skipping event for parent %s: %s", ownerRef.Kind, err)
```

Reply:
Hey @mateiidavid , I have already did the changes on my local development environment, rebased the commits , but i am facing problems in pushing the code :

```
adarsh@chiku:~/myfiles/OpenSource/linkerd2$ git push -u origin webhook-bug 
To https://github.com/Adarsh-jaiss/linkerd2.git
 ! [rejected]            webhook-bug -> webhook-bug (non-fast-forward)
error: failed to push some refs to 'https://github.com/Adarsh-jaiss/linkerd2.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```

and to fix this whenever i am running `git pull origin webhook-bug`, a merge conflict is getting arised and it isn't getting resolved, i have been trying to fix it since morning!

This happened last time as well in the previous PR, if you could help me out, that would be very kind of you :)

Reply:
mm have you tried `git pull origin webhook-bug --strategy=recursive -X theirs`?

Reply:
I tried this and merged the changes in my codebase, and still now working!

Is there any other way?